src/ParallelAlgorithms/Events/ObserveFields.hpp

```diff
@@ -142,11 +161,13 @@ class ObserveFields<VolumeDim, ObservationValueTag, tmpl::list<Tensors...>,
 
   explicit ObserveFields(const std::string& subfile_name,
                          const std::vector<std::string>& variables_to_observe =
-                             {db::tag_name<Tensors>()...},
+                             VariablesToObserve::default_value(),
+                         std::optional<Mesh<VolumeDim>> interpolation_mesh = {},
                          const Options::Context& context = {})
       : subfile_path_("/" + subfile_name),
         variables_to_observe_(variables_to_observe.begin(),
-                              variables_to_observe.end()) {
+                              variables_to_observe.end()),
```

Comment:
not sure why this is taken by const-ref and copied. Could you change it to taken-by-value and `std::move` it instead? You can also `std::move` the mesh. (If clang-tidy says the move is trivial, just silence it)

Reply:
I can do that. This will also make the check for duplicates redundant (and impossible as we are moving from std::vector into std::unordered_set so duplicate information is lost). I will  remove the corresponding unit test failure case.

Reply:
Oh I didn't realize this is stored as an `unordered_set` but the option parses as a `vector`. Probably best to leave as-is then (unless we added option-parsing for `unordered_set` at some point)

Reply:
I suggest you don't bother with this for this PR

Reply:
ha, too late. the implementation I have now should still be option parsable though as the argument is still a stringvector